docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Export PostgreSQL logs from YugabyteDB Anywhere universes
+headerTitle: Export audit logs
+linkTitle: Export logs
+description: Export universe logs to third-party tools.
+headcontent: Export PostgreSQL logs from your universe
+badges: ea
+menu:
+  preview_yugabyte-platform:
+    identifier: universe-logging
+    parent: alerts-monitoring
+    weight: 70
+type: docs
+---
+
+Export YSQL database logs to third-party tools for security monitoring, to build operations and health dashboards, troubleshooting, and more. You can export the following types of logs:
+
+<!-- Database query logging. This is the standard [PostgreSQL logging](https://www.postgresql.org/docs/11/runtime-config-logging.html) facility. Using these settings you can log query statements and errors.-->
+- Database audit logging. Using the [PostgreSQL Audit Extension](https://www.pgaudit.org/#) ([pgaudit](https://github.com/pgaudit/pgaudit/blob/1.3.2/README.md)), the audit log provides the exact database transactions, which is a compliance requirement for government, financial, or ISO certifications.
```

Comment:
@ddhodge @Sahith02 
do we need to mention the pgaudit version as users cannot change it and the supported logging parameters are documented in this article? what do we do for other extensions / plugins that we support?